### ISSUE TITLE
New scrub feature?

### ISSUE THREAD

**Issue by ddevz** [blind-oracle/zabbix-zfs#17]
Hey, I saw you have a check for "is it currently scrubbing?".  However, I think a more important one would be "It's been X days since your last scrub", which would verify that automatic monthly scrubbing is still working

**Comment by blind-oracle** [comment]
Yeah, it might be useful, but parsing stuff like `scrub repaired 0B in 08:40:44 with 0 errors on Mon Mar 27 19:44:38 2023` and all its variations looks shitty. ZFS is quite hard to query in an automated way.

**Comment by maufimauf** [comment]
I added a prototype trigger on the key `zfs.pool.scrub["{#POOL_NAME}"]` with a macro `{$ZFS_SCRUBBED_WITHIN_TIME}`. Setting to zero disables & resolves the trigger.

However, doing it this way, the time counter will only start after the last scrub ended (state moved to 0), it serves our purpose and I guess also the requested one. On long scrub times the trigger might be a bit delayed for short scrub intervals. I can't think of any use-case though.

```
uuid: db6f84b3aa6d4d698e7c70355e0951f8
expression: 'max(/Template_ZFS/zfs.pool.scrub["{#POOL_NAME}"],{$ZFS_SCRUBBED_WITHIN_TIME})<1 and {$ZFS_SCRUBBED_WITHIN_TIME}>0'
name: 'Pool "{#POOL_NAME}" no scrub started after {$ZFS_SCRUBBED_WITHIN_TIME}'
priority: WARNING
```
```
macro: '{$ZFS_SCRUBBED_WITHIN_TIME}'
value: 32d
description: 'Time since last scrub before trigger warning (0=off)'
```
I know 6.2 is out-dated but it's the only one I got right now, so feel free to merge it to the other versions.
[template_zfs_zbx6.2.zip](https://github.com/blind-oracle/zabbix-zfs/files/12431906/template_zfs_zbx6.2.zip)
